### ISSUE TITLE
Confusing release status

### ISSUE THREAD

**Issue by flack** [browserstate/history.js#269]
I've been watching this project for a while now, because I'm planning to integrate it into a framework I'm working on. While I can figure out the necessary steps for making it work, I'm kind of puzzled by the current release status. The repo's front page says the current release is 1.7.1 from October 2011, there is also a 1.8 tag which points to a branch where the readme says it's version 1.8a4 (https://github.com/browserstate/history.js/tree/1.8). both branches seem to have received commits since 2011, so I was just wondering where I should base my work on and what the general plan is. Will the dev branch be merged into master at some point, is it considered stable already, or will the two versions be developed in parallel?
This might seem a minor issue, but it sure is confusing and might steer new contributors or implementers in the wrong direction. So any clarification you can offer would be greatly appreciated.
